src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -791,7 +791,7 @@ private TypedConstant GetDefaultValueArgument(ParameterSymbol parameter, Attribu
                 CheckFeatureAvailability(syntax.ArgumentList, MessageID.IDS_FeatureCallerArgumentExpression, diagnostics);
                 parameterType = GetSpecialType(SpecialType.System_String, diagnostics, syntax);
                 kind = TypedConstantKind.Primitive;
-                defaultValue = syntax.ArgumentList.Arguments[argumentIndex].Expression.ToString();
+                defaultValue = syntax.ArgumentList.Arguments[argumentIndex].Expression.ToStringWithoutTrivia();
```

Comment:
>ToStringWithoutTrivia [](http://example.com/codeflow?start=87&length=21)

I think the LDM resolution was misinterpreted. I believe it requires no changes to implementation. Here is a quote: "we go from the start of real C# executable code to the end of the expression, not including any leading or trailing trivia." Nowhere does it say that anything should be done to the trivia in the middle.

Reply:
@AlekseyTs So it should simply be `syntax.WithoutTrivia().ToString()` (i.e, ignore leading and trailing trivia, but keep the trivia of inner nodes)?

Reply:
>  So it should simply be syntax.WithoutTrivia().ToString() (i.e, ignore leading and trailing trivia, but keep the trivia of inner nodes)?

As I said, I believe no change is needed here. The value should be the expression without its leading and trailing trivia.

Reply:
> So it should simply be syntax.WithoutTrivia().ToString()

That is already equivalent to syntax.ToString() @Youssef1313.   Thanks!

Reply:
@AlekseyTs Currently all trivia are included.

Reply:
> Currently all trivia are included.

What test demonstrates that?

Reply:
> What test demonstrates that?

It's my bad memory 😄 Sorry!